app/serializers/keywords_serializer.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class KeywordsSerializer
```

Comment:
@malparty Can we have only one `KeywordSerializer` and use Conditional attributes https://github.com/jsonapi-serializer/jsonapi-serializer#conditional-attributes